### ISSUE TITLE
Main windows doesn't remember the vertical position

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#156]
```
What steps will reproduce the problem?
1. Start Pino.
2. Hide the main window either using notification area icon or close control.
3. Unhide the main windows using the notification area icon.

What is the expected output? What do you see instead?
I expect the window to remember its last position. But it only remember the
horizontal position. Vertical position changes, sliding down each time
until it reaches a panel.

What version of the product are you using? On what operating system?
Pino 0.2.4 (RPM) on Fedora 12 (x86)

Please provide any additional information below.
- Was is earlier versions too.
- I also have Compiz installed and running
- Fedora 12 system is up to date
- I use GNOME Global Menu

```

Original issue reported on code.google.com by `gave...@gmail.com` on 28 Mar 2010 at 6:02

Attachments:
- [pino_screenshot.png](https://storage.googleapis.com/google-code-attachments/pino-twitter/issue-156/comment-0/pino_screenshot.png)

**Comment by GoogleCodeExporter** [comment]
Original comment by `tro...@gmail.com` on 28 Mar 2010 at 9:18
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Closed by 4801486fa0 revision.
```

Original comment by `tro...@gmail.com` on 28 Mar 2010 at 2:58
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
That was fast. Thanks. :)
```

Original comment by `gave...@gmail.com` on 28 Mar 2010 at 4:44
